Detectors/TRD/base/include/TRDBase/Digit.h

```diff
@@ -46,11 +46,13 @@ class Digit : TimeStamp
   void setRow(int row) { mRow = row; }
   void setPad(int pad) { mPad = pad; }
   void setADC(ArrayADC const& adc) { mADC = adc; }
+  void setTime(double eventtime) { mTime = eventtime; }
```

Comment:
You should rebase to dev your PR. The current Digit class inherits TimeStamp from the common dataformats.

Reply:
So, you will have to rebase to dev after this PR https://github.com/AliceO2Group/AliceO2/pull/2840.

Reply:
@jolopezl  Yes